test/jdk/java/net/httpclient/SendResponseHeadersTest.java

```diff
@@ -83,19 +84,20 @@ public void testSend() throws Exception {
 
     static class TestHandler implements HttpHandler {
         public void handle(HttpExchange exchange) throws IOException {
-            exchange.sendResponseHeaders(200, 0);
+            try (InputStream is = exchange.getRequestBody();
+                 OutputStream os = exchange.getResponseBody()) {
 
-            OutputStream os = exchange.getResponseBody();
-            try {
-                IOException io = expectThrows(IOException.class,
-                        () -> exchange.sendResponseHeaders(200, "failMsg".getBytes().length));
-                System.out.println("Got expected exception: " + io);
-            } catch (Throwable t) {
-                // unexpected exception thrown, return error to client
-                t.printStackTrace();
-                os.write(("Unexpected error: " + t).getBytes());
-            } finally {
-                os.close();
+                is.readAllBytes();
+                exchange.sendResponseHeaders(200, 0);
+                try {
+                    IOException io = expectThrows(IOException.class,
+                            () -> exchange.sendResponseHeaders(200, "failMsg".getBytes().length));
```

Comment:
It would be clearer to use a constant value here: either 0, or some meaningless value > 0 and < 16

Reply:
replaced with 0. See https://github.com/openjdk/jdk/pull/1014/commits/8ff6d9b4fcbc0f57f136ff0e7df0d371664badb2